src/components/Book.js

```diff
@@ -1,21 +1,41 @@
+/* eslint-disable import/no-extraneous-dependencies */
 import React from 'react';
 import PropTypes from 'prop-types';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
 
 function Book({ book, removeBook }) {
   return (
     <div>
-      <div className="book-section" key={book.item_id}>
+      <div className="book-section" key={book.itemId}>
         <div className="book-innerSection">
           <span className="book-category">{book.category}</span>
-          <p className="book-title">{book.title}</p>
+          <h5 className="book-title">{book.title}</h5>
           <span className="book-author">{book.author}</span>
           <div className="btn-container">
             <button type="button" className="btn">Comment</button>
             <div className="line" />
-            <button type="button" className="btn" onClick={() => removeBook(book.item_id)}>Remove</button>
+            <button type="button" className="btn" onClick={() => removeBook(book.itemId)}>Remove</button>
             <div className="line" />
             <button type="button" className="btn">Edit</button>
+
+          </div>
+        </div>
+        <div className="progress-bar">
+          <div className="progress-container">
+            <div style={{ width: 70, height: 70 }}>
+              <CircularProgressbar value={1} />
+            </div>
           </div>
+          <div className="text">
+            <span className="Percent-Complete">2%</span>
+            <span className="Percent-Complete1">Completed</span>
```

Comment:
- [ ] 👏 Great work on adding the circular progress bar! However, to bring the text of this progressbar in line with the design, please consider updating the text's color, font styling, and other visual aspects to match the design as closely as possible. Attention to these details will ensure that your project's visual consistency is maintained. Keep up the good work! 👍🎨🔄

| Your | Design |
| --- | --- |
|  ![image](https://github.com/Obimbo07/book-library/assets/55840999/5a60a7c0-4789-41b4-bb62-7782a2da2354)| ![image](https://github.com/Obimbo07/book-library/assets/55840999/ba334704-ce60-43f4-b470-a41df4b6fab2)|